py/server/tests/test_table.py

```diff
@@ -809,22 +824,38 @@ def test_slice(self):
         with self.assertRaises(DHError):
             rt = t.slice(3, 2)
 
+    def test_rollup(self):
+        test_table = empty_table(100)
+        test_table = test_table.update(
+            ["grp_id=(int)(i/5)", "var=(int)i", "weights=(double)1.0/(i+1)"]
+        )
+        with self.assertRaises(DHError) as cm:
+            rollup_table = test_table.rollup(aggs=self.aggs)
+        self.assertRegex(str(cm.exception), r".+ is not supported for rollup")
```

Comment:
It seems like you should be testing each "bad" agg individually, instead of all at once.